scripts/init_letsencrypt.sh

```diff
@@ -33,10 +32,7 @@ docker compose run --rm --entrypoint "\
 
 echo
 
-echo "### Copy the certificate and key to their final destination ..."
-cp ${CONFIG_PATH}/certbot/conf/live/${QFIELDCLOUD_HOST}/fullchain.pem ${CONFIG_PATH}/nginx/certs/${QFIELDCLOUD_HOST}.pem
-cp ${CONFIG_PATH}/certbot/conf/live/${QFIELDCLOUD_HOST}/privkey.pem ${CONFIG_PATH}/nginx/certs/${QFIELDCLOUD_HOST}-key.pem
-echo
+chmod 755 "$CONFIG_PATH/nginx/99-autoreload.sh"
```

Comment:
I believe if we keep it executable in the git repo, that should be sufficient?

Reply:
It is intentionally like this, so that the autoreload is not executed as long as it is not necessary. 

If it should always be executed, even if not necessary, it would be sufficient to make it executable in git.

Reply:
Ah I see.
In this case wouldn't it be more transparent if it was done with something more "visible" like renaming this file?

Reply:
This would be possible but would be way more complicated and i don't know if it is worth this. 

If we copy it with the script we should be aware of:
- We have to delete the folder that is created by docker-compose
- Should add the file to .gitignore
- We still need to apply the permissions

I can change it to your preference, but i would be fine with the simple chmod.

Reply:
@suricactus what's your opinion on this?

Reply:
I am in favor of keeping things explicit. I do not see any significant risk by reloading nginx every X hours, so I would prefer to commit the file with the `+x` permission.

Reply:
Done